packages/api-explorer/__tests__/lib/is-auth-ready.test.js

```diff
@@ -78,16 +78,44 @@ describe('isAuthReady', () => {
         apiKeyScheme: '',
         oauthDiff: '',
       }),
-    ).toBe(false);
+    ).toBe(true);
 
     expect(
       isAuthReady(operation, {
         oauthScheme: '',
         apiKeyScheme: 'key',
         oauthDiff: '',
       }),
+    ).toBe(true);
+
+    expect(
+      isAuthReady(operation, {
+        oauthScheme: '',
+        apiKeyScheme: '',
+        oauthDiff: '',
+      }),
+    ).toBe(true);
+  });
+
+  it('should return false if required security scheme is not provided', () => {
+    const operation = oas2.operation('/required-and-or-security', 'post');
+
+    expect(
+      isAuthReady(operation, {
+        oauthScheme: 'bearer',
+        apiKeyScheme: '',
+        oauthDiff: '',
+      }),
     ).toBe(false);
 
+    expect(
```

Comment:
In the assertion should returns false, but here returns true

Reply:
No it's fine, maybe the test description is not clear enough. This assertions verifies that false is returned when the required scheme is not provided (for the test it's the `apiKey` scheme that is provided in this assertion)

Reply:
I've splitted the test in two to make it clearer

Reply:
Yes, it was the assertion wrong